packages/excalidraw/element/routing.ts

```diff
@@ -1034,3 +991,30 @@ const getBindPointHeading = (
     elementsMap,
     origPoint,
   );
+
+const getHoveredElements = (
```

Comment:
Oh, looks like I got my answer for the binding order, assuming you need to determine hovered elements by iterating them top-down. Is this the only reason?

Reply:
Yes! In the case of the history and remote change application it doesn't matter, but I don't want to break the convention of z-index ordered elementsMap as a parameter when later down the road we modify the code and this remains forgotten.